modules/runner/src/main/ts/api/Main.ts

```diff
@@ -1,28 +1,36 @@
-import { Runner } from '../core/Runner';
-import { TestData } from '../core/TestTypes';
-import { Ui } from '../ui/Ui';
+import { Global } from '@ephox/bedrock-common';
+import * as Patches from '../core/Patches';
 import { UrlParams } from '../core/UrlParams';
 import { makeSessionId } from '../core/Utils';
 import { Callbacks } from '../reporter/Callbacks';
 import { Reporter } from '../reporter/Reporter';
+import { Runner } from '../runner/Runner';
```

Comment:
I was wondering if you were going to use something like this and replace the runner page: https://mochajs.org/#running-mocha-in-the-browser

Do you have some comments in the readme or somewhere on the approach to integrating mocha?

Reply:
That's effectively what's happening under the hood, we just don't use the mocha UI and keep our own. I wasn't keen to change this, as a lot of positioning tests might potentially fail since Mocha dumps a whole heap of content while the tests are running. Additionally, we would have lost the ability to have our "restart", "skip" or "retry" buttons.